packages/peregrine/lib/talons/Breadcrumbs/useBreadcrumbs.js

```diff
@@ -5,8 +5,8 @@ import { useQuery } from '@apollo/react-hooks';
 const sortCrumbs = (a, b) => a.category_level > b.category_level;
 
 // Generates the path for the category.
-const getPath = (path, suffix) => {
-    if (path) {
+const getPath = (category_url_path, suffix) => {
+    if (category_url_path) {
```

Comment:
Huh, was this just a bug? The arg was previously named `path`, so was `category_url_path` always undefined before?

Also, for variables and args we define, we generally use camelcase (`categoryUrlPath`).

Reply:
done